Tribler/Test/test_as_server.py

```diff
@@ -173,8 +173,8 @@ def checkReactor(self, phase, *_):
                                      "Listening ports left on the reactor during %s: %s" % (phase, reader))
 
         # Check whether the threadpool is clean
-        tp_items = [item for item in reactor.getThreadPool().working if item]
-        self.assertEqual(len(tp_items), 0, "Threadpool instances left: %s" % reactor.getThreadPool().working)
+        self.assertEqual(len(reactor.getThreadPool().working), 0,
+                         "Threadpool instances left: %s" % reactor.getThreadPool().working)
```

Comment:
The contents of `reactor.getThreadPool().working` will be a list of `None` values, this won't provide any extra insights for the error message (if anything, it will probably confuse the programmers). For this reason, you should probably not print the contents of it.